config/intellij-java-google-style.xml

```diff
@@ -0,0 +1,598 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Isn't it better to configure intellij to use prettier and spotless? (https://www.jetbrains.com/help/idea/prettier.html)

Reply:
I have not used the prettier plugin for intellij before. 
I'm not sure how to add it for Java.
When we added the google code style, we had no problem when we edited the code with intellij. 
We even saw that the spottlessCheck method was successful build without warning.

Reply:
We use spotless to define the java formatting in ANET. It is best to find a way to instruct intellij to use spotless for that. If that it is not possible, and you still would like to manually maintain a intellij code formatting spec, please do not check it in the main codebase - as it is IDE specific and will result in having to maintain that file on every upgrade of spotless.

Reply:
Thank you for feedback , I've removed this file from the package